apps/sms/services/README.md

```diff
@@ -0,0 +1,91 @@
+# The layers in the Messages application
+
+## Terms explanation
+
+There are 3 layers whose interfaces managed by the Bridge (aka [threads.js](https://github.com/gaia-components/threads/)):
+
+* Services define high-level semantics to access the model. They are the only
```

Comment:
> Ok, I didn't want to mean that this was directly called as in a direct javascript call. Definitely this uses bridge.client.
I consider the bridge library as a technical help; so when I say "direct" I mean "without any functional intermediary object".

I see, so the last thing to clarify here, do you want to see level of abstraction around ```bridge.client``` (like I'm trying to do with ActivityClient) or you think we should use ```bridge.client``` directly from view (like direct call in JS :))?

> Yeah, I'd rather do that unless we see it's a bad idea.

Sounds ok for me.

> Not sure I follow :) Wrapper is another type of bridge-managed service except it lives in a window. And it's really a "technical" artefact while managers are "functional" objects, that's why I give different names.

Yep I meant this, will keep it in my head as "service-shim_for_api_that_is_not_available_in_worker" then :) For the name, I'm fine with ```Wrapper``` or ```Shim```.